### ISSUE TITLE
The color of circles is incorrect

### ISSUE THREAD

**Issue by FireCubez** [ggez/ggez#650]
**Describe the bug**
Whenever I draw a circle, it just does **not** want to be the right color.

**To Reproduce**
This is my draw code
```rust
fn draw(&mut self, _controller: &mut Controller, ctx: &mut Context) -> GameResult<()> {
    graphics::clear(ctx, Color::from_rgba(201, 56, 56, 255));
    let size = graphics::drawable_size(ctx);
    let outc = graphics::Mesh::new_circle(
        ctx,
        graphics::DrawMode::fill(),
        mint::Point2 { x: size.0 / 2.0, y: size.1 / 2.0 },
        self.syncer().player_size() + 10.0,
        0.1,
        Color::from_rgba(160, 32, 32, 255),
    )?;
    let inc = graphics::Mesh::new_circle(
        ctx,
        graphics::DrawMode::fill(),
        mint::Point2 { x: size.0 / 2.0, y: size.1 / 2.0 },
        self.syncer().player_size(),
        0.1,
        Color::from_rgba(206, 33, 33, 255)
    )?;
    graphics::draw(ctx, &outc, graphics::DrawParam::default())?;
    graphics::draw(ctx, &inc, graphics::DrawParam::default())?;
    graphics::present(ctx)
}
```

**Expected behavior**
I expected the correct colors to be drawn.

**Screenshots or pasted code**
![image](https://user-images.githubusercontent.com/39168965/60814817-a4967180-a196-11e9-8ee2-a1cc8cf7a1b2.png)

The background is cleared correctly, however both the circles have incorrect colors.

**Hardware and Software:**
 - ggez version: 0.5.0-rc.2
 - OS: Windows 10

**Comment by icefoxen** [comment]
This is probably an sRGB issue, we might be applying the conversion twice at some point or something.  Try setting `WindowSetup::srgb(false)` in your `ContextBuilder` and see if that gets the right behavior?

**Comment by FireCubez** [comment]
I actually was about to open this issue but then I found a solution and I think I accidentally still created the issue. Anyways, the problem is an inconsistency between `graphics::clear` and drawing. `graphics::clear` appears to accept a color that is sRGB-encoded while the latter expects *linear* RGB. The fix is just to convert the sRGB colors to linear RGB when passing to a mesh. The inconsistency still needs to at least be documented somewhere, or better fixed.

**Comment by icefoxen** [comment]
Yeah, since `Color` is defined to be sRGB but our shaders operate in linear color space, we need to do the conversion from sRGB to linear when we create a `Mesh` or such.  Thank you!

**Comment by icefoxen** [comment]
Resolved by #656
